db/migrate/20220217154815_create_comment.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.integer :Authorid
+      t.integer :Postid
+      t.text :Text
+      t.date :UpdatedAt
+      t.date :CreatedAt
```

Comment:
For Ruby's best practices, Kindly use `snake_case` for methods and variables.
`:Authorid` should be `:author_id`
`:UpdatedAt` should be `updated_at`
`:CreatedAt` should be `created_at`